acp/ideas_module.php

```diff
@@ -152,6 +153,42 @@ public function main($id, $mode)
 
 		if ($submit)
 		{
+			if ($this->request->is_set_post('ideas_forum_setup') && (int) $this->config['ideas_forum_id'])
```

Comment:
This probably should not be part of submit but its own condition, otherwise this not only makes the change but also tries to submit the other fields